components/Common/ShellBox/index.module.scss

```diff
@@ -0,0 +1,75 @@
+.shellBox {
+  background-color: var(--black10);
+  border-radius: 0.4rem;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  font-family: var(--mono);
+  padding: 0 0 var(--space-48) var(--space-16);
+  position: relative;
+
+  code {
+    color: var(--pink5);
```

Comment:
On `nodejs.dev`, we have extra styles that allow some optional colouring... Any reason you didn't migrate those styles? They would need to be migrated differently.

**But pretty much you could do:**

`strong` and each `span` inside `code` to have some right margin, and make maybe certain elements to have different colour...

**So if I had:**

```tsx
<strong>$</strong> <span>choco</span> install nvm
```

It could create similar results from the images below.

![image](https://user-images.githubusercontent.com/12037269/233779376-c47a972c-b947-466d-9868-fc64d05f79dc.png)
![image](https://user-images.githubusercontent.com/12037269/233779324-56df129b-ceb7-4e7f-b767-3aef8ee38199.png)

Reply:
the "second style" from component named "OSpannel" eg: "MacosPannel".
So I don't thing we should migrate se second "style"

Reply:
I didn't get your answer, sorry?

Reply:
I did not migrate the additional styles, because they are part of the `OSpannel` component [Example](https://github.com/nodejs/nodejs.dev/tree/main/src/components/DownloadComponents/InstallTabs).

So I don't think we should add this style with the migration of the componen

Reply:
If you say so. Then sure... We can always migrate this at a later time.